packages/node_modules/action-chain/package.json

```diff
@@ -15,18 +15,25 @@
 		"clean": "rimraf es lib coverage",
 		"typecheck": "tsc --noEmit",
 		"test": "jest",
-    "test:watch": "jest --watch --updateSnapshot --coverage false",
+		"test:watch": "jest --watch --updateSnapshot --coverage false",
+		"prepare": "npm run build",
 		"prebuild": "npm run clean",
 		"postbuild": "rimraf {lib,es}/**/__tests__",
 		"posttest": "npm run typecheck"
 	},
-	"keywords": [ "action", "flow" ],
-	"files": [ "lib", "es" ],
+	"keywords": [
+		"action",
+		"flow"
+	],
+	"files": [
+		"lib",
+		"es"
+	],
 	"dependencies": {
 		"@types/node": "^10.5.1",
-		"eventemitter3": "^3.1.0",
-    "tslib": "^1.9.3",
-    "color": "^3.0.0"
+		"color": "^3.0.0",
+		"emittery": "^0.4.1",
```

Comment:
No idea whats going on [on travis](https://travis-ci.org/cerebral/overmind/builds/406027127#L707) but I added `emittery` to the dependencies here.